appium-cookies/index.js

```diff
@@ -0,0 +1,6 @@
+// transpile:main
+
+import {default as sample} from './lib/sample';
+
+export {sample};
```

Comment:
We have also standardized on spaces around the curly braces in imports and exports. So `import { default as sample } from './lib/sample''`.